eng/test_tools.txt

```diff
@@ -13,16 +13,20 @@ bandit==1.6.2
 
 # locking packages defined as deps from azure-sdk-tools or azure-devtools
 pytoml==0.1.21
-readme-renderer[md]==25.0
+readme-renderer[md]==29.0
 pyOpenSSL==19.1.0
 json-delta==2.0
 ConfigArgParse==1.2.3
 six==1.14.0
-pyyaml==5.3.1
+pyyaml==5.4.0
 packaging==20.4
 wheel==0.34.2
 Jinja2==2.11.2
 
+# pinning these to resolve CG alerts
+urllib3==1.26.5
```

Comment:
If you pin urllib3, you need to pin requests too, otherwise you will get into trouble since requests is pinning urllib3:
https://github.com/psf/requests/blob/1e5fad7433772b648fcbc921e2a79de5c4c6be8b/setup.py#L48

Reply:
Hmm.

This makes sense. I believe the actual issue is coming when we run `mindependency` testing on `azure-core` (or anyone who takes dep on requests really). 

We don't actually want to raise the minimum version right? So it needs to be a test-only override. :|

This is lame, we may want to get an exception of doing the simple pin like I do above isn't enough.